src/ir/local-graph.h

```diff
@@ -51,18 +57,41 @@ struct LocalGraph {
   // for a param.
   //
   // Often there is a single set, or a phi or two items, so we use a small set.
+  //
+  // If this LocalGraph is in lazy mode then this may perform the computation
+  // of sets at this time. As a result, the returned Sets from this function
+  // should be considered invalidated by other calls to getSets (the same as if
+  // you read from a std::unordered_map and then modified that map). The const
+  // version of this method has no such issue, of course.
```

Comment:
This is not necessary. Inserting or removing items in a `std::unordered_map` does not move or otherwise invalidate any of the other items in the map.

Reply:
Oh, cool, I didn't know that!

Indeed I see now in the [docs](https://en.cppreference.com/w/cpp/container/unordered_map)

> References and pointers to either key or data stored in the container are only invalidated by erasing that element, even when the corresponding iterator is invalidated.